openpos-server-core/src/main/java/org/jumpmind/pos/server/config/SecuredAutoConfiguration.java

```
@@ -150,7 +150,7 @@ private static Path ensuredPath(final Path path) throws IOException {
             try {
                 Files.setPosixFilePermissions(parent, PosixFilePermissions.fromString(DEFAULT_SECURITY_DIR_PERMS));
             } catch (UnsupportedOperationException ex) {
-                log.warn("unable to set least perms on {} (non-POSIX-compliant file system)", parent);
+                log.warn("unable to set least perms on {} (non-POSIX-compliant file system)", parent, ex);
```

Well should be a seperate PR, but really don't want to swallow exceptions